From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Tue, 31 Dec 2013 01:31:41 +0100
Subject: debian-multiarch

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7d808cc..430caeb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -152,7 +152,7 @@ endif()
 
 set(BIN_INSTALL_DIR "bin")
 set(LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)")
-set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}")
+set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}/${CMAKE_LIBRARY_ARCHITECTURE}")
 set(INCLUDE_INSTALL_DIR "include/gammaray")
 if(WIN32)
   set(PLUGIN_INSTALL_DIR "plugins")
