From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Sat, 7 Feb 2015 13:48:20 +0100
Subject: disable-qt4-objectvisualizer

---
 plugins/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index 5e5feff..660b7e1 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -10,7 +10,6 @@ add_subdirectory(webinspector)
 
 if(Qt5Core_FOUND)
   add_subdirectory(translatorinspector)
-else()
   add_subdirectory(objectvisualizer)
 endif()
 
