From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Wed, 4 Jan 2012 21:23:17 +0100
Subject: find-qt-webkit

---
 CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5501542..03bccdb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,7 @@ add_definitions(-DPROGRAM_NAME=\"${Prog_NAME}\")
 include(CheckCXXCompilerFlag)
 
 set(QT_MIN_VERSION "4.7.0")
-find_package(Qt4 REQUIRED)
+find_package(Qt4 COMPONENTS QtWebKit REQUIRED)
 
 if(CMAKE_BUILD_TYPE MATCHES "^[Rr]elease$")
   add_definitions(-DQT_NO_DEBUG_OUTPUT)
-- 
