From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Thu, 26 Dec 2013 10:14:12 +0100
Subject: find-qt5-webkit-5.1

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 46e2c60..eaa31d2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -97,7 +97,7 @@ if(Qt5Core_FOUND)
     Location # workaround for 5.1 webkit that tries to find a qtlocation with a matching version if none (with any version) has been found before...
     Designer
   )
-  find_package(Qt5WebKitWidgets 5.0.2 QUIET) # 5.0.[01] has a code generation bug in uic that fails to generate proper code for webkitwidget classes
+  find_package(Qt5WebKitWidgets QUIET)
 
   include("cmake/ECMQt4To5Porting.cmake")
   # only needed for Qt <= 5.1 and/or some older CMake versions (exact combination unknown unfortunately)
