From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Tue, 29 May 2012 09:09:26 +0200
Subject: gcc-4.7-build-fix

---
 hooking/abstractfunctionoverwriter.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hooking/abstractfunctionoverwriter.h b/hooking/abstractfunctionoverwriter.h
index 20d0669..b4671b4 100644
--- a/hooking/abstractfunctionoverwriter.h
+++ b/hooking/abstractfunctionoverwriter.h
@@ -25,6 +25,8 @@
 #include <QList>
 #include <QString>
 
+#include <unistd.h>
+
 #if defined(_M_X64) || defined(__amd64) || defined(__x86_64)
 #define ARCH_64
 #elif defined(_M_IX86) || defined(__i386__)
