Description: adapt to Qt 5.4 API changes
Origin: upstream, https://github.com/KDAB/GammaRay/commit/a10e2103f09cb471
Bug: https://launchpad.net/bugs/1395646
Last-Update: 2015-01-13

--- a/plugins/quickinspector/quickinspector.cpp
+++ b/plugins/quickinspector/quickinspector.cpp
@@ -358,9 +358,14 @@
   const QSGTextureProvider *provider = m_source->textureProvider();
   Q_ASSERT(provider);
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 4, 0)
   const QQuickShaderEffectTexture *texture =
     qobject_cast<QQuickShaderEffectTexture*>(provider->texture());
   Q_ASSERT(texture);
+#else
+  const QSGLayer *texture = qobject_cast<QSGLayer*>(provider->texture());
+  Q_ASSERT(texture);
+#endif
 
   QOpenGLContext *ctx =
     QQuickItemPrivate::get(m_source)->sceneGraphRenderContext()->openglContext();
