From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Thu, 10 Jan 2013 20:16:29 +0100
Subject: use-empty-rpath

---
 CMakeLists.txt |    2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4bbee30..1e62b19 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,8 +30,6 @@ set(GAMMARAY_VERSION "${GAMMARAY_VERSION_MAJOR}.${GAMMARAY_VERSION_MINOR}.${GAMM
 set(GAMMARAY_VERSION_STRING "${GAMMARAY_VERSION}")
 set(GAMMARAY_PROBE_SOVERSION "${GAMMARAY_VERSION_STRING}")
 
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 
 enable_testing()
