From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Thu, 10 Jan 2013 20:16:29 +0100
Subject: use-empty-rpath

---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 92e1955..6cbe7de 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,8 +36,6 @@ set(GAMMARAY_VERSION_STRING "${GAMMARAY_VERSION}")
 set(GAMMARAY_SOVERSION "${GAMMARAY_VERSION}")
 set(GAMMARAY_PLUGIN_VERSION "2.0")
 
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-
 enable_testing()
 
 if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
